docs/building-a-node-with-ldk/handling-events.md

```diff
@@ -0,0 +1,72 @@
+# Handling Events
+
+LDK requires that you handle many different events throughout your app's life cycle. You can learn more by reading about our event-driven [architecture](/overview/architecture.md).
+
+To start handling events in your application, run:
+
+<CodeSwitcher :languages="{rust:'Rust', kotlin:'Kotlin', swift:'Swift'}">
+  <template v-slot:rust>
+
+```rust
+use lightning::util::events::{Event};
+
+// In the event handler passed to BackgroundProcessor::start
```

Comment:
Looks like `BackgroundProcessor` was removed from the guide. We'll want to have a section on it as it drives everything. Probably somewhere in the introduction.

Reply:
Fixed in https://github.com/lightningdevkit/lightningdevkit.org/pull/156/commits/43b12ca9540aa078a38c34afc1f56a0b20911fec